postcss.config.js

```diff
@@ -10,3 +10,11 @@ module.exports = {
     })
   ]
 }
+
+module.exports = {
+    plugins: ['stylelint-scss'],
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rikai-academy/booking_tours_01/pullRequest?prid=7829742)
```suggestion
    plugins: ["stylelint-scss"],
```